components/BUILD.gn

```diff
@@ -61,13 +63,46 @@ test("brave_components_unittests") {
       "//ui/gfx",
     ]
   }
+
+  # No components should depend on Chrome.
+  # Since //chrome depends on //extensions, we also only assert_no_deps on
+  # extensions targets.
+  assert_no_deps = [
+    "//brave/browser/*",
```

Comment:
there are currently several `//brave/app` deps that are not easily removed so leaving that out for now